news/6858.feature

```diff
@@ -0,0 +1 @@
+Warn about packages not found by pip show.
```

Comment:
You want to surround ``pip show`` with double backticks, before and after, since it is command syntax.

Reply:
Done.